framework/lib/theme/components/table/index.ts

```diff
@@ -25,5 +35,47 @@ export const Table: ComponentMultiStyleConfig = {
         borderTopWidth: 'xs',
       },
     }),
+    striped: (props) => {
+      const { colorScheme: c = 'gray' } = props
```

Comment:
Can we execute a new token for this one? Since it will be nice to have "themed" stripes in the future. So a component token for the table can be changed if changing theme in the future. What do you think?